### PR TITLE
[Community][minor]: Added langchain_version while calling discover API

### PR DESCRIPTION
**Pull request by dristysrivastava**
- **Description:**  Added langchain version while calling discover API during both ingestion and retrieval
- **Issue:** NA
- **Dependencies:** NA
- **Tests:** NA
- **Docs** NA

### CODE REVIEW

File: libs/community/langchain_community/chains/pebblo_retrieval/base.py [hunk 1]
```diff
@@ -326,6 +327,7 @@ def _get_app_details(  # type: ignore
             framework=framework,
             chains=chains,
             plugin_version=PLUGIN_VERSION,
+            langchain_version=langchain_community.__version__,
```
Comment (eyurtsev) [inline libs/community/langchain_community/chains/pebblo_retrieval/base.py:330]: Not sure if you care, but this is the community version. You probably want the core version as well .

langchain-community
langchain
langchain-core

are all different packages

Comment (dristysrivastava) [reply libs/community/langchain_community/chains/pebblo_retrieval/base.py]: Yes, @eyurtsev updated this

**Comment by eyurtsev** [comment]
cc @Raj725 could confirm?

**Comment by Raj725** [comment]
> cc @Raj725 could confirm?

@eyurtsev These changes are pending internal review. We'll mark this PR as ready once the review is complete. Going forward, we'll ensure internal reviews are done before raising a PR. Thanks!

**Review by Raj725 — APPROVED** [review]

**Comment by dristysrivastava** [comment]
@eyurtsev Please review

**Comment by srics** [comment]
@eyurtsev we intentionally want to grab `langchain_community` to better support our install base, as that is the primary pkg with pebblo components. We also grab & show `langchain` pkg version as well (https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/utilities/pebblo.py#L268). We are good with this change from pebblo maintainers.

Pls review, thx!

**Comment by ccurme** [comment]
@dristysrivastava sorry for the delay here. There are some merge conflicts; if you resolve these and ping me I'll review.

**Comment by dristysrivastava** [comment]
> @dristysrivastava sorry for the delay here. There are some merge conflicts; if you resolve these and ping me I'll review.

@ccurme Resolved the merge conflicts. Please review

**Review by ccurme — APPROVED** [review]

**Pull request merged**